packages/api/src/external/carequality/command/cq-directory/search-cq-directory.ts

```diff
@@ -57,13 +60,24 @@ export async function searchCQDirectoriesAroundPatientAddresses({
 export async function searchCQDirectoriesByRadius({
   coordinates,
   radiusInMeters,
+  mustHaveXcpdLink = false,
 }: {
   coordinates: Coordinates[];
   radiusInMeters: number;
+  mustHaveXcpdLink?: boolean;
 }): Promise<CQDirectoryEntryModel[]> {
   const orgs: CQDirectoryEntryModel[] = [];
 
   for (const coord of coordinates) {
+    // Building the WHERE clause conditionally
+    let whereClause = `earth_box(ll_to_earth (${coord.lat}, ${coord.lon}), ${radiusInMeters}) @> point
```

Comment:
I know these coordinates come from AWS Location Service, but to make this safer, can we move this to [Bind Parameters](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#c85966880dbb409fb169da526918c8cf)?

Or maybe ticket it and add it to the sharpen the saw with the `security` label?